src/main/scala/pa/model.scala

```diff
@@ -1,11 +1,17 @@
 package pa
 
-import org.joda.time.{Interval, DateTime, LocalDate}
+import java.time.{LocalDate, LocalDateTime, ZoneOffset}
+
+import org.feijoas.mango.common.collect.{Range => MangoRange}
+
+
 
 case class Error(message: String)
 
 case class Season(competitionId: String, seasonId: String, name: String, startDate: LocalDate, endDate: LocalDate){
-  lazy val interval: Interval = new Interval(startDate.toDateTimeAtStartOfDay, endDate.toDateTimeAtStartOfDay)
+
+  implicit val localDateOrdering: Ordering[LocalDateTime] = Ordering.by(_.toEpochSecond(ZoneOffset.UTC))
+  lazy val interval = MangoRange.closed(startDate.atStartOfDay(), endDate.atStartOfDay())
```

Comment:
Do you know if that's used in MAPI / dotcom?

Reply:
it's not used in MAPI (iit's internal). I'll check dotcom

Reply:
It isn't used externally. It's a joda construct with no equivalent in jave time

Reply:
So if it's not used externally, and if my quick github search is correct, it's only used for the purpose of unit tests. In which case if we drop that attribute we can drop an extra dependency?

Reply:
Plan. Seems Reasonabl